_scripts/instruction-widget/install.js

```diff
@@ -95,6 +97,7 @@ module.exports = function(context) {
 
   debian_install = function() {
     template = "debian";
+    context.devuan = context.distro == "devuan"
```

Comment:
Won't this make every debian install devuan? Since it's in the body of the debian_install function the context.devuan flag will be truethy meaning that everyone will get the devuan backports instructions